### PR TITLE
Add /forums command

### PR DESCRIPTION
**Pull request by Bryan-0**

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1432,6 +1432,11 @@ var commands = exports.commands = {
 		this.sendReplyBox("<a href=\"https://www.smogon.com/sim/staff_list\">Pokemon Showdown Staff List</a>");
 	},
 
+	forums: function () {
```
Comment (AustinXII) [inline chat-plugins/info.js]: you are forgetting (target, room, user)

Comment (Bryan-0) [inline chat-plugins/info.js]: Lol i forget that.

**Comment by Bryan-0** [comment]
Done.

**Comment by Bryan-0** [comment]
I'm going to squash this commits, but first I want the opinion of @Zarel.

**Comment by Zarel** [comment]
I'll let Joim decide.

**Comment by Bryan-0** [comment]
Ok.

**Comment by Joimer** [comment]
This can be useful since those forums have relevant PS information and the appeals. But yeah, squash.

**Comment by TheImmortal** [comment]
Needs more é

**Comment by Bryan-0** [comment]
You mean on "Pokémon"? @TheImmortal

**Comment by TheImmortal** [comment]
Yes, what else would I mean.

**Comment by kotarou3** [comment]
aka Poke a cute mon (Pok&eacute;mon)

**Comment by Bryan-0** [comment]
I squash the commits, and I fix everything. @Joimer or @Zarel.

**Comment by Joimer** [comment]
But the acute e is not used anywhere or mostly anywhere. Just see some lines above: `Pokemon Showdown Staff List`

We should stick to one style. I say we go all acute, so I'm merging this in and we should then work on that acute everywhere else.

**Comment by Bryan-0** [comment]
Alright.

**Comment by Zarel** [comment]
I would rather not use Unicode in code files for now. If it's escaped as `&eacute;` and `\u00e9`, that's fine, though.

**Pull request merged**